packages/eslint-plugin-query/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@tanstack/eslint-plugin-query",
+  "version": "4.13.0",
+  "description": "ESLint plugin for TanStack Query",
+  "author": "Eliya Cohen",
+  "license": "MIT",
+  "repository": "tanstack/query",
+  "homepage": "https://tanstack.com/query",
+  "funding": {
+    "type": "github",
+    "url": "https://github.com/sponsors/tannerlinsley"
+  },
+  "main": "build/lib/index.js",
+  "scripts": {
+    "typecheck": "tsc",
+    "build": "tsup --minify",
+    "dev": "tsup --watch --sourcemap",
```

Comment:
if we decide to use [rollup to bundle as cjs](https://rollupjs.org/guide/en/#--bundleconfigascjs), then we should ensure it'll generate source maps when working in development mode for easier debugging (breakpoints, etc).

Reply:
I'm fine with keeping tsup for now if it works.